compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -2234,8 +2248,7 @@ fn add_local_native_libraries(
                 }
             }
             NativeLibKind::RawDylib => {
-                // FIXME(#58713): Proper handling for raw dylibs.
-                bug!("raw_dylib feature not yet implemented");
+                // Ignore RawDylib for now, we'll handle these later.
```

Comment:
I think the following would be clearer:

> // Ignore RawDylib here, they are handled separately in `linker_with_args()`.

Otherwise, it could be interpreted as "this is not yet implemented".

Reply:
Fair point.